getting-started-tutorial/tutorial-1-MLRun-basics.ipynb

```diff
@@ -0,0 +1,1890 @@
+{
```

Comment:
<p>better to say: we use the <code>%nuclio env</code> or the <code>%nuclio cmd</code> ﻿.</p><p>especially after writing <code># nuclio</code>  annotations a few times before.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/demos/pull/73/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>